crates/context/src/lib.rs

```diff
@@ -217,6 +218,23 @@ impl ContextManager {
                 )
             }
 
+            let context_id: IdentityContextId = context.id.rt().expect("infallible conversion");
+            let member_id = identity_secret
+                .public_key()
+                .rt()
+                .expect("infallible conversion");
```

Comment:
you won't need the `IdentityContextId` import if you localized the transmutation to the calling position.

Reply:
Updated